packages/visx-shape/yarn.lock

```diff
@@ -0,0 +1,293 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
we avoid committing lock files in specific packages, and just have them in the monorepo root. could you remove these and then re-run `yarn` in the root just to make sure the root lock file is good?

Reply:
Ah good call, done.